### DISCUSSION TITLE
.eslintrc.yml: declare and use a variable

### DISCUSSION THREAD

**Discussion by fauxbytes** [eslint/eslint#17283] [Q&A] [answered]
Hi.
How can I define a variable in eslint's config (yaml format), and then reference that variable for configuring rules, within the same yaml config.

Sth like:
```
# define FOO=<some regex>
rules: {
    'my/plugin': [warn, { 'src/**', FOO }]
}
```

Thanks

**Comment by aladdin-add** [comment, marked as answer]
not sure if it's possible in yaml, my recommendation is to use the js configs if you want to do so.
